api/src/main/resources/db/changelog/local/changelog-2.22.1-module-github-token-id.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+            http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+    <changeSet id="2-22-1-3" author="stanley.zhang@ityin.net">
+        <addColumn tableName="module">
+            <column name="github_app_token_id" type="varchar(36)" />
+        </addColumn>
+        <addForeignKeyConstraint baseTableName="module" baseColumnNames="github_app_token_id"
```

Comment:
I have a question about this part.

is this going to allow nullable values for modules that are not using a github app @stanleyz right ?

for example, in other scripts we used something like the following:

       <addColumn tableName="module" >
            <column name="ssh_id" type="varchar(36)">
                <constraints nullable="true" foreignKeyName="fk_module_ssh" references="ssh(id)"/>
            </column>
        </addColumn>

Reply:
The default value of nullable is true for the `addForeignKeyConstraint`, but all good with it explicitly declared. 

I updated the column line, it's a shame that the `addForeignKeyConstraint` doesn't support `nullable` and yet `constraints` in `column` doesn't support cascade settings.